src/csharp/Grpc.Core.Tests/ChannelTest.cs

```diff
@@ -123,5 +123,16 @@ public async Task ChannelBaseShutdownAsyncInvokesShutdownAsync()
             // check that Channel.ShutdownAsync has run
             Assert.AreEqual(ChannelState.Shutdown, channel.State);
         }
+
+        [Test]
+        public void CompositeCredentialsWithInsecureThrow()
+        {
+            var compositeCredentials = ChannelCredentials.Create(
+                ChannelCredentials.Insecure,
+                CallCredentials.FromInterceptor((context, metadata) => Task.CompletedTask));
```

Comment:
`/var/local/git/grpc/src/csharp/Grpc.Core.Tests/ChannelTest.cs(132,77): error CS0117: 'Task' does not contain a definition for 'CompletedTask' [/var/local/git/grpc/src/csharp/Grpc.Core.Tests/Grpc.Core.Tests.csproj]`. This is due to the .NET45 target.

You can use https://github.com/grpc/grpc/blob/fc302e37f2c1ec1db112bafd26427d65301ad9dd/src/csharp/Grpc.Core/Utils/TaskUtils.cs#L32 instead.